### ISSUE TITLE
Q: Where is current documentation on TraceViewer UI capabilities

### ISSUE THREAD

**Issue by 2ooom** [catapult-project/catapult#4212]
Hello,

First of all thank you for maintaining and improving the project. I've read [documentation ](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview#) on trace format but I still struggle to understand how to implement in my traces same visualizations as I've seen in the examples `tracing\test_data\`.

Particularly I'm trying to add a `CPU Usage` track and something like `Interactions` in Chrome Buttor example. Reading the code I found classes `Slice`, `ThreadSlice`, `ThreadTimeSlice` that have `cpuDuration` filed used by `CpuUsageAuditor` to generate the cpu track.

I was wondering if there is more documentation related to this things?

Thank you

**Comment by fmeawad** [comment]
I don't think we have more documented related to those things, you can also check the `catapult/tracing/tracing/ui/tracks/cpu_usage_track_test.html` for an example of `CPU Usage` track.

@benshayden might know more about the `Interactions` track.

**Comment by 2ooom** [comment]
Thank you @fmeawad

**Comment by PeterHuewe** [comment]
Did you have any success? I can't get similar results either :/
